src/__tests__/components/component.test.tsx

```diff
@@ -20,7 +20,7 @@ describe('Component Test Suite', () => {
             element: <SearchInput />
         }]);
         render(<RouterProvider router={router} />);
-        
+
```

Comment:
Maybe I can get with you to compare prettier configs. Your's is correct, not sure why my editor does not catch these.